measures/apps/measuresdata/measuresdata.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+#
+# measuresdata.sh
+# Must be executable. Call it with all defaults or arguments for measuresdata
+# It is a test script to enable writing python (or other) versions
+
+echo Calling measuresdata $*
+measuresdata $*
```

Comment:
In this case it will make no difference (the arguments cannot have spaces); however I will change it. Thanks for the comment.